generators/server/templates/src/test/java/package/web/rest/_LogsResourceIntTest.java

```diff
@@ -19,10 +19,11 @@
 package <%= packageName %>.web.rest;
 
 <%_ if (databaseType === 'cassandra') { _%>
-import <%= packageName %>.AbstractCassandraTest;
+import <%=packageName%>.AbstractCassandraTest;
```

Comment:
I personally think its better to have space as it makes it more readable and i normally add space whenever i come across these